apprclient.go

```diff
@@ -263,7 +262,7 @@ func (c *Client) do(ctx context.Context, req *http.Request, v interface{}) error
 
 		return nil
 	}
-	b := backoff.NewExponential(backoff.ShortMaxWait, backoff.ShortMaxInterval)
+	b := backoff.NewMaxRetries(3, 1*time.Second)
```

Comment:
@rossf7 @tomahawk28 I changed this because the unit tests would just hang forever. Do you think this is an issue?

Reply:
If this works, I think it's fine.